docs/deployment.md

```diff
@@ -0,0 +1,93 @@
+# Deployment
+
+## Deploying locally
+
+Your function can run locally either by starting it through Visual
```

Comment:
If it depends on the Invoker or a custom invoker, seems evident to us but might not be to everyone.

Reply:
Done.